neo/idlib/StrStatic.h

```diff
@@ -97,39 +97,39 @@ class idStrStatic : public idStr
 	{
 		buffer[ 0 ] = '\0';
 		SetStaticBuffer( buffer, _size_ );
-		idStr::operator=( b );
+		idStrStatic::operator=( b );
 	}
 	
 	ID_INLINE	explicit idStrStatic( const char c ) :
 		idStr()
 	{
 		buffer[ 0 ] = '\0';
 		SetStaticBuffer( buffer, _size_ );
-		idStr::operator=( c );
+		idStrStatic::operator=( c );
```

Comment:
I don't think this will work (neither here nor anywhere else where you replaced `idStr::operator=` with `idStrStatic::operator=`)

maybe take a look at https://github.com/OpenTechEngine/OpenTechBFG/pull/136 which aims to fix the same compiler errors, I already commented on all the problems there.